src/dev/build/args.ts

```diff
@@ -50,6 +51,8 @@ export function readCliArgs(argv: string[]) {
       rpm: null,
       deb: null,
       'docker-images': null,
+      'docker-push': false,
+      'docker-tag-qualifier': '',
```

Comment:
If we're checking for the presence of a qualifier, this should probably default to null.

Reply:
Pushed with bd82d5c0f4fbad8bf9f0c34f82c6939d951aed7d